assets/css/v2/busway/departures/header.scss

```diff
@@ -0,0 +1,19 @@
+.departures-header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  font-family: Inter, sans-serif;
+  font-size: 56px;
+  font-weight: 700;
+  line-height: 67.77px;
+  text-transform: uppercase;
+  letter-spacing: 3px;
```

Comment:
**question:** Do we have shared classes for these kinds of typographic styles?

Reply:
Things like font sizes and line heights tend to be fairly specific to each combination of component and screen type. I'm not aware of anything other than the mixins in `fonts.scss`, though some of this could be put in there.

Reply:
All set! f15e3fdbe3245191620e04967ef37cca05dd6294